Panelled propagation example

larshinueber opened the pull request:
Brief example that showcases how to setup a simple panelled radiation target model

DominicDirkx commented:
@larshinueber I completely lost track of this example, my apologies! It looks really good, one small request: could you also add a plot with the two total accelerations? That will put the behaviour and order of magnitude of the difference into perspective better (and, could you modify the m/s^2 axis label to make the '2' a superscript?)

larshinueber commented:
I also forgot about this 😅
I added the total accelerations instead of the difference and a plot with the RSW components of both position and acceleration differences. Removed the Keplerian element plots, as I don't think it gives all too much insight in this case
